std/range/package.d

```diff
@@ -1106,6 +1106,16 @@ if (Ranges.length > 0 &&
                     }
                     return result;
                 }
+
```

Comment:
Because I needed a specialized logic for flattening of the `chain`-tree as mentioned below.